sailfishusermanager.h

```diff
@@ -72,6 +73,7 @@ private slots:
 
 private:
     bool checkAccessRights(uint uid_to_modify);
+    bool setUserUUID(uint uid);
```

Comment:
I don't see definition for this and I don't exactly see the reason to have this, so maybe this line is a leftover and it should be removed?

Reply:
Yes, maybe some old code. I remove this.